### DISCUSSION TITLE
I love Stellarium!

### DISCUSSION THREAD

**Discussion by kishorekumar62** [Stellarium/stellarium#3232] [Show and tell]
Over the last one year, I have created [a set of Stellarium scripts](https://www.youtube.com/@Ahargana) that explain the astronomy of the _Hindu calendar_ in great detail.

When I started on this endeavor, I was not entirely sure if it is was even possible to simulate each and every calendric concept in Stellarium. As I proceeded forward, I found the Stellarium scripting API to be so wonderfully comprehensive and well thought out that I was indeed able to simulate all calendric concepts. And the one time I needed something new (ecliptic elongation of the Moon), I requested for it  (#1786) and it was implemented immediately by @gzotti. And when I got down to explaining the precession of the equinoxes, the presence of StelMovementMgr.setFlagLockEquPos was like "did somebody read my mind"?

A big Thank You to all of you who contribute to this software! You guys are absolutely fantastic.

**Comment by gzotti** [comment]
You are welcome. Over the last years I also implemented the Calendars plugin, including a few Indian calendars. I hope they are useful!
